kuma/spam/models.py

```diff
@@ -13,3 +13,33 @@ class Meta:
         get_latest_by = 'created'
         ordering = ('-created',)
         abstract = True
+
+
+class AkismetSubmission(models.Model):
+    """
+    Stores the submissions to Akismet that are trickered by trusted writers.
+
+    This shall be used as a log of sorts to make sure we know what the data
+    has been reported to Akismet as spam or ham.
+    """
+    SPAM_TYPE, HAM_TYPE = 'spam', 'ham'
+    TYPES = (
+        (SPAM_TYPE, _('Spam')),
+        (HAM_TYPE, _('Ham')),
+    )
+    sender = models.ForeignKey(settings.AUTH_USER_MODEL)
+    sent = CreationDateTimeField(
+        _('sent at'),
+        db_index=True,
+    )
+    type = models.CharField(
```

Comment:
The thesaurus has some suggestions: ``category``, ``sort``, ``classification``, ``genre``, ``species``, ``grade``, ``analysis``, ``assignment``. But, ``type`` is OK with me.